model/http-common.yaml

```diff
@@ -54,6 +54,11 @@ groups:
           Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
       - ref: http.route
         requirement_level: opt_in
+        brief: >
```

Comment:
there is already brief and note in registry, if we need to update it, it makes sense to make it explicit there

Reply:
cc @lmolkova  - the registry could acknowledge both http client and server frameworks for this attribute.

Reply:
I'd prefer to keep current brief/note for server side. If we change brief/note in the registry, we'd need to override them  for both.

Since http.route on the client is a rare/future case, I'd prefer to just override it for the client side without changing the common one. We can always refactor yaml later, markdown  is the source of truth.